packages/cli/src/commands/bundle/buildBundle.ts

```diff
@@ -4,25 +4,45 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
+// @ts-ignore - no typed definition for the package
 import Server from 'metro/src/Server';
-
+// @ts-ignore - no typed definition for the package
 import outputBundle from 'metro/src/shared/output/bundle';
 import path from 'path';
 import chalk from 'chalk';
-import type {CommandLineArgs} from './bundleCommandLineArgs';
-import type {ConfigT} from 'types';
+import {CommandLineArgs} from './bundleCommandLineArgs';
+// @ts-ignore FIXME after converting types
+import {ConfigT} from 'types';
```

Comment:
```suggestion
import {Config} from '@react-native-community/cli-types';
```
Remove the above ts-ignore as well